doc/example-keywords.py

```diff
@@ -128,7 +128,7 @@ def get_all_keywords():
     skip = set(text.splitlines())
 
     all_keywords = Counter()
-    for d in (cantera_root / "interfaces/cython/cantera/examples").glob("**"):
+    for d in (cantera_root / "samples/python").glob("**"):
```

Comment:
```suggestion
    for d in (cantera_root / "samples" / "python").glob("**"):
```
to avoid Windows path separator problems.